.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,36 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - master
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
This is an old version of the action, please update otherwise you'll get strange issues and possible vulnerabilities: https://github.com/actions/checkout
```suggestion
      - uses: actions/checkout@v3
```